src/themes/teams/components/Icon/svg/icons/video-camera-emphasis.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import cx from 'classnames'
+import { TeamsProcessedSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
+      <g>
+        <path
+          className={cx('ui-icon__outline', classes.outlinePart)}
```

Comment:
@miroslavstastny I'm using the poc you did in #364. i manually added this to a couple icons for this toolbar, but we will need to get this added to all icons. are you planning on doing this?

Reply:
We agreed that #364 is the way how we want to do it but nobody is currently working on it.
Created an issue #716 (feel free to take it :-)